test_settings.py

```diff
@@ -129,6 +129,7 @@ def root(*args):
 ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE = 512   # Enterprise logo image size limit in KB's
 
 ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES = ['audit', 'honor']
+ENTERPRISE_COURSE_ENROLLMENT_VERIFIED_MODES = ['verified', 'professional']
```

Comment:
Will have to make a PR on `edx/configuration` for this and add it to `edx/edx-platform` as well, I believe. Not even sure if this is correct, though!